.travis.yml

```diff
@@ -1,3 +1,3 @@
-language: java
+language: groovy
 sudo: false
-script: mvn clean verify
+script: mvn clean test
```

Comment:
Does ```verify``` cause trouble which we avoid with ```test```?

Reply:
Seems like we have now a ```mvn clean install``` followed by a ```mvn clean test```.
See our travis build #8. This is not how it should be, I suppose.